### PR TITLE
Broken link removal

### PR DESCRIPTION
**Pull request by ixodie**
Link doesn't work.  Removing.

### CODE REVIEW

**Comment by anubha-v-ardhan** [comment]
@ixodie 
The link works for me. Can you recheck please :)

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -258,8 +258,6 @@ Multus supports all [reference plugins](https://github.com/containernetworking/p
 
 [VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
 
-[NSX-T Container Plug-in (NCP)](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS/PaaS platforms such as Pivotal Container Service (PKS) and OpenShift.
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/networking.md:261]:
```suggestion
[NSX-T Container Plug-in (NCP)](https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/com.vmware.nsxt.ncp_kubernetes.doc/GUID-52A92986-0FDF-43A5-A7BB-C037889F7559.html) provides A CNI plugin that you can use to integrate your Kubernetes cluster with VMware NSX-T network fabric.

```

Hyperlinking to the specific page that mentions a CNI plugin feels reasonable to me.

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -258,8 +258,6 @@ Multus supports all [reference plugins](https://github.com/containernetworking/p
 
 [VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/networking.md:259]: How about:
```suggestion
[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for cloud or bare-metal infrastructures.
```

Comment (ixodie) [reply content/en/docs/concepts/cluster-administration/networking.md:259]: Does it make sense to have this paragraph if there is nothing else in the entry?

Comment (sftim) [reply content/en/docs/concepts/cluster-administration/networking.md:259]: Hyperlinking to the specific page that mentions a CNI plugin [feels reasonable to me](https://github.com/kubernetes/website/pull/30093/files#r729784622).

Comment (sftim) [reply content/en/docs/concepts/cluster-administration/networking.md:259]: The comment you're replying to links to my [previous suggestion](https://github.com/kubernetes/website/pull/30093/files#r729784622) @ixodie, videlicet:
- link to https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/com.vmware.nsxt.ncp_kubernetes.doc/GUID-52A92986-0FDF-43A5-A7BB-C037889F7559.html

I still recommend that; I'm also open to considering other URIs.

**Comment by reylejano** [comment]
Closing this PR as @anubha-v-ardhan and myself find that the link works

/close

**Pull request closed without merging**